bloom/filter_seqs_from_biom.py

```diff
@@ -0,0 +1,70 @@
+import biom
+import argparse
+import sys
+
+
+def trim_seqs(seqs, length = 100):
+    """
+    Trims the sequences to a given length
+
+    Parameters
+    ----------
+    seqs: generator of skbio.Sequence objects
+
+    Returns
+    -------
+    generator of skbio.Sequence objects
+        trimmed sequences
+    """
+    for seq in seqs:
+        yield seq[:length]
+
+def remove_seqs(table, seqs):
+    """
+    Parameters
+    ----------
+    table : biom.Table
+       Input biom table
+    seqs : generator, skbio.Sequence
+       Iterator of sequence objects to be removed from the biom table.
+
+    Return
+    ------
+    biom.Table
+    """
+    filter_seqs = set([str(s) for s in seqs])
+    bloom_filter = lambda v, i, m: i in filter_seqs
+    table.filter(bloom_filter, axis='observation')
```

Comment:
XD

Yeah, I was really confused when I first heard that term.

Not sure about the filter.  We want to keep sequences that aren't blooms right?